website/docs/r/vapp_org_network.html.markdown

```diff
@@ -37,6 +44,12 @@ The following arguments are supported:
 * `org_network_name` - (Optional; *v2.7+*) An Org network name to which vApp network is connected. If not configured, then an isolated network is created.
 * `is_fenced` (Optional) Fencing allows identical virtual machines in different vApp networks connect to organization VDC networks that are accessed in this vApp. Default is false.
 * `retain_ip_mac_enabled` - (Optional) Specifies whether the network resources such as IP/MAC of router will be retained across deployments. Configurable when `is_fenced` is true.
+* `reboot_vapp_on_destroy` - (Optional; *v3.9+*) **VCD 10.4.1+** API **prohibits removal of vApp
+  network from a powered on vApp**. This field can be used to power off vApp during vApp network
+  removal. It will power on the vApp (if it was so) after removing network. (default `false`)
```

Comment:
Please rephrase this sentence, the "(if it was so)" is not clear.

Reply:
Added additional context

Reply:
Thanks! I'm reading the new version:

> It will power on the vApp (if it was not powered off before the operation) **after** removing network.

Shouldn't "after" be "before"?

Reply:
> Shouldn't "after" be "before"?

`This field can be used to power off vApp during vApp network removal. It will power on the vApp (if it was not powered off before the operation) after removing network.`

The complete meaning of this sentence is meant to state that during vApp network removal the following will happen:
1. vApp is powered off (if it was not)
2. vApp network is removed
3. vApp is powered back on (if it was not powered off before the operation)

That comment should highlight:

Reply:
OK, the confusion comes because of the "not" (double logical negation with "not" and "off"):

> It will power on the vApp (if it was **not** powered off before the operation)

I'm thinking how to simplify this. What if we change this:

`This field can be used to power off vApp during vApp network removal. It will power on the vApp (if it was not powered off before the operation) after removing network.`

->

```Set to `true` to power off the vApp during vApp network removal. If the vApp's original state was powered on, it will be powered back on after removing the network.```

Reply:
Sounds good for me

Reply:
Changed